src/batch_inclusion/mod.rs

```diff
@@ -83,7 +86,17 @@ impl BatchInclusion {
         account_id: AccountId,
         payouts: Vec<Payout>,
     ) -> Result<Vec<PayoutWithInclusionEstimate>, BatchInclusionError> {
-        let queues = self.payout_queues.list_by_account_id(account_id).await?;
+        // let queues = self.payout_queues.list_by_account_id(account_id).await?;
+    let mut has_next_page = true;
+        let mut queues = Vec::new();
+        while has_next_page {
+            let paginated_queues = self
+                .payout_queues
+                .list_for_account_id_by_id(account_id, Default::default(), Default::default())
```

Comment:
this will always get the first page - need to use `paginated_queues.into_next_query()` after `drain(..)`ing the entities.